### DISCUSSION TITLE
Static files 404 not found

### DISCUSSION THREAD

**Discussion by Mobbbb** [nuxt/nuxt#24678] [Questions] [answered]
I use below config start my pm2 server
```
module.exports = {
    apps: [
        {
            name: 'SSR',
            port: '3003',
            exec_mode: 'cluster',
            instances: 'max',
            script: './.output/server/index.mjs',
        }
    ]
}
```
And then add nginx config:
```nginx
location / {
    proxy_pass http://127.0.0.1:3003;
}
```
When I access https://x.x.x.x:80 through nginx, the site of html can render normally, and status code is 200 too.But the static files which in `_nuxt` folder can't access, and return 404 code.
![1](https://github.com/nuxt/nuxt/assets/30068563/3d267022-e241-44af-8fdf-b2d6f8b59e65)
![3](https://github.com/nuxt/nuxt/assets/30068563/784aecd2-8763-4675-b04f-9279922c5802)

If I accessed http://x.x.x.x:3003 which is my nuxt3 server port directly, all files return 200 code, everything is displayed normally.
![2](https://github.com/nuxt/nuxt/assets/30068563/24641eca-0c66-483b-8dd1-b5bdab1959fe)
![4](https://github.com/nuxt/nuxt/assets/30068563/c2244f80-1556-458a-80a0-351e67637d70)

Is there any wrong with my nginx config?Or should I add any other config in my nginx?

Server environment：
```
Ubuntu 20.04.6 LTS
pm2: 5.2.0
node: v18.19.0
nginx version: nginx/1.18.0 (Ubuntu)
```

Dev environment:
```
nuxt: "^3.8.0"
```

**Comment by xjccc** [comment]
pm2 host: '0.0.0.0' ?

**Reply by Mobbbb** [reply]
It's pm2 list and server port.Should I set the host explicitly?
![5](https://github.com/nuxt/nuxt/assets/30068563/956c9a1b-b4ef-4323-bfcc-925121fa410d)

**Reply by xjccc** [reply]
pm2.config

```
apps: [
 {
      name: 'ssr',
      exec_mode: 'cluster',
      script: 'node-server/server/index.mjs',
      env: {
        PORT: 3003,
        HOST: '0.0.0.0'
      }
    }
]
```
nginx config

```
location / {
    proxy_pass http://127.0.0.1:3003/;
}
```

**Reply by Mobbbb** [reply]
I've tried.But it seems doesn't work.
I can access the html file through nginx, just can't access static files of `_nuxt` folder, so I think there are some wrong with nginx config.

**Reply by xjccc** [reply]
could you use pm2 at your computer to access the html and static files?

**Comment by Mobbbb** [comment, marked as answer]
I've solved this problem.
I found that as long as I delete the following code, the program can run normally.
```nginx
location ~ .*\.(ttf|woff|webp|gif|jpg|jpeg|bmp|png|ico|txt|js|css)$ {
        expires max;
        if_modified_since off;
        add_header Last-Modified "";
        etag off;
}
```
But I don't know what the specific problem is
